swifter/swifter.py

```diff
@@ -124,6 +128,7 @@ def rolling(self, window, min_periods=None, center=False, win_type=None, on=None
         return Rolling(self._obj, self._npartitions, self._dask_threshold, self._progress_bar, **kwds)
 
     def _wrapped_apply(self, func, axis=0, broadcast=None, raw=False, reduce=None, result_type=None, args=(), **kwds):
+        # TODO: `reduce` is a standard keyword in Python, prefer an alternate name.
```

Comment:
reduce is a parameter in pandas.DataFrame.apply, so I don't think we can take this out